hack/check-state.sh

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env bash
```

Comment:
This file appears to duplicate the same file in this PR https://github.com/kubevirt/hyperconverged-cluster-operator/pull/336. I think 336 should be reviewed and merged first and then rebase this PR on top of it. Or did you have a different merging process in mind?

Reply:
done.